host-bmc/host_pdr_handler.cpp

```diff
@@ -840,6 +840,15 @@ void HostPDRHandler::processHostPDRs(mctp_eid_t /*eid*/,
         pldm::hostbmc::utils::updateEntityAssociation(
             entityAssociations, entityTree, objPathMap, oemPlatformHandler);
 
+        if (this->isfirstgetPDR)
```

Comment:
this logic should be moved completely to the getPDR function in platform.cpp right ? - we need to do this before even getting the PDR's from host (not when the last record handle from host is null)

Reply:
> this logic should be moved completely to the getPDR function in platform.cpp right ? - we need to do this before even getting the PDR's from host (not when the last record handle from host is null)

Ack